src/ophys_etl/transforms/mesoscope_2p/conversion_utils.py

```diff
@@ -22,12 +23,15 @@ def dump_dict_as_attrs(h5fp, container_name, data):
 
 def volume_to_h5(h5fp, volume, dset_name="data", page_block_size=None,
                  **h5_opts):
+    if isinstance(h5fp, str):
+        f = h5py.File(h5fp, "w")
```

Comment:
this file is never closed

Reply:
how is this resolved?
just because github marks it as "outdated" does not mean it is resolved.
thee file opened on line 27 is never closed.

Reply:
That code doesn't exist anymore. This comment is pointing to an outdated version.

Reply:
ok, my bad. I reloaded, but it was still showing for me in `Files Changed` for a while,